CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKVisualProcessor.java

```diff
@@ -99,7 +99,7 @@ void addTask(final CVKRenderableUpdateTask task) {
     @Override
     protected final void destroyCanvas() {
         //TODO_TT: is this needed?
-        System.out.println("GLVisualProcessor.destroyCanvas called");        
+        System.out.println("CVKVisualProcessor.destroyCanvas called");        
```

Comment:
Just delete this print to System.out which we never see anyway.  Not sure what the cleanup needs to be here.  Want to look into it?

Reply:
Deleted old comments, added Destroy() to canvas and removed component listener